### ISSUE TITLE
High CPU usage (25% on 5500u) even when in background 

### ISSUE THREAD

**Issue by palamosteliaro** [project-sbc/Handheld-Control-Panel#24]
I don't think this is normal, the app uses 25% CPU constantly even when its minimized at the taskbar. Any of the features I should disable to lower the CPU usage? If its supposed to use so much CPU its not even worth using.

**Comment by NJinsei** [comment]
Yeah the same things happening to me as well on a intel i7 6300hq

**Comment by project-sbc** [comment]
Are either of you on discord?

Best Regards,
Chris

WeChat: ProjectSBC
QQ: 1619238952
________________________________

**Comment by palamosteliaro** [comment]
> 

Not really using it, but I could open it to help. What should I do?

**Comment by project-sbc** [comment]
I’d like to see your cpu usage , I’m looking at 0.2% to 0.7% usage with mine, cpu is not limited and all cores are active. I could imagine if your tdp is really low or you disabled most of your cores the usage could skyrocket. There could be an issue with threads being generated

Best Regards,
Chris

WeChat: ProjectSBC
QQ: 1619238952
________________________________

**Comment by palamosteliaro** [comment]
![3](https://github.com/project-sbc/Handheld-Control-Panel/assets/9744496/1108bca9-e34e-4d06-b805-dbf60651abd6)

First screenshot on a 5500u (Hp Envy 360) and the second on a desktop with a 5800x3d. I know your software is not intended for desktops, but I thought to try it and see if there is high CPU usage on my desktop too.
On both I just opened HCP, set everything to max and left it open. The high CPU usage was there before I touched anything and stayed there indefinitely.

**Comment by twotreeszf** [comment]
Me too, about 13% high CPU usage on GPD Win4

**Comment by dmomen** [comment]
Same here. Really great tool, but this issue discards it. Constantly loading CPU.

**Comment by project-sbc** [comment]
Overhaul coming, cpu load and ram usage drop significantly

Best Regards,
Chris
________________________________
